server/appServices.js

```diff
@@ -4,6 +4,7 @@ const androidVariantCRD = require('./android-variant-crd.json');
 const iosVariantCRD = require('./ios-variant-crd.json');
 const mobileSecurityServiceCRD = require('./mobile-security-crd.json');
 const { debounce } = require('lodash');
+const _ = require('lodash');
```

Comment:
lodash is already imported above this.

```suggestion
const { debounce, includes } = require('lodash');
```

Reply:
The reason for the importing lodash as `_` is as follows. When `includes` was directly import there was strange things happening. I was getting odd results. As I was exploring issues I felt that the importing the namespace for lodash was safer. I will keep this in mind for the final PR